arcade/utils.py

```diff
@@ -16,6 +16,17 @@ def lerp_vec(v1: Vector, v2: Vector, u: float) -> Vector:
     )
 
 
+def lerp_angle(start_angle: float, end_angle: float, u: float):
+    """Linearly interpolate between two angles in degrees, following the shortest path."""
```

Comment:
Considering this method specifically lerps using the shortest path in a circle, the method should probably be named to relfect this? It could cause confusion.

Reply:
I thought it was pretty clear, as if you were lerping an angle, this is pretty likely the behaviour you'd want. Happy to take suggestions though.

Reply:
What about `lerp_angle_shortest`? I just fear people will use this for angles in general.

Reply:
I'm happy to change it to that - though I'm not sure why people using this for angles in general would necessarily be a bad thing...

Reply:
Then let's just keep it as is.